easybuild/easyconfigs/b/BBMap/BBMap-38.75-GCC-8.3.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'MakeCp'
+
+name = 'BBMap'
+version = '38.75'
+
+homepage = 'https://sourceforge.net/projects/bbmap/'
+description = """BBMap short read aligner, and other bioinformatic tools."""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['28cab0aa43780f5857be9223e50c31ea']
```

Comment:
@soheil-soltani Please make this a SHA256 checksum (`eb --inject-checksums` is your friend here).